core_java/twitter/src/main/java/ca/jrvs/apps/twitter/model/Coordinates.java

```diff
@@ -0,0 +1,50 @@
+package ca.jrvs.apps.twitter.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import java.util.List;
+
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "coordinates",
+        "type"
+})
+
+public class Coordinates {
+
+    @JsonProperty("coordinates")
+    private List<Float> coordinates;
```

Comment:
List is OK, but you can consider using Array for better performance, as the length is fixed here

Reply:
thanks!